domain/network/state/linklayer.go

```diff
@@ -280,6 +283,16 @@ func (st *State) getSubnetGroups(ctx context.Context, tx *sqlair.TX) (subnetGrou
 		return nil, errors.Errorf("running subnets query: %w", err)
 	}
 
+	// Filter out placeholder subnet for orphan IPs
+	subs = filter(subs[:0], subs, func(s subnet) bool {
```

Comment:
You will need to do this filtering where you have recruited it instead of here.

There are actually some /32 addresses that we do want to match. We have manual clouds that use BGP routing, and have a /32 added to the loopback address. We discover these addresses (and add the subnets) gradually when manual machines are added. 

If the secondary address moves from one NIC to another, it needs to find the matching subnet on the way back in.